index.html

```diff
@@ -59,13 +114,90 @@ <h4 class="canopy">canopy</h4>
+          <h2>Multi-Post Stories</h2>
+          <div class="tags-holder">
+            <h4 class="canopy">canopy</h4>
+            <div class="dot"></div>
+            <span class="tag">Back End Dev</span>
+            <div class="dot"></div>
+            <span class="tag">2022</span>
+          </div>
         </div>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="skills">
+          <li class="btn-skill">html</button>
+          <li class="btn-skill">css</button>
+          <li class="btn-skill">javaScript</button>
+        </ul>
         <button class="view-project">See project</button>
-      </section> -->
-    </main>
-  </body>
-</html>
+      </div>
+    </section>
+    <section class="about-section">
+      <h2 class="about-self">About Myself</h1>
+        <p class="about-me">
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+          work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a class="connect" href="#">let's connect</a>
+        <ul class="social-icons">
+          <li> <a href="#"><img class="icon" src="./imgs/twitter.svg" alt="twitter" /> </a>
+          </li>
+          <li>
+            <a href="#"> <img class="icon" src="./imgs/linked-in.svg" alt="linked-in" /> </a>
+          </li>
+          <li> <a href="#"> <img class="icon" src="./imgs/just-m.svg" alt="just-m" /> </a> </li>
+          <li> <a href="#"> <img class="icon" src="./imgs/github.svg" alt="github" /> </a></li>
+          <li> <a href="#"><img class="icon" src="./imgs/finger.svg" alt="finger" /> </a> </li>
+        </ul>
+        <button class="view-project resume">Get my resume</button>
+        <section class="main-skills">
+          <div class="languages"> 
+              <h4>languages</h4>
+            
+              <img  class="arrow" width="14px" height="8px" src="./imgs/arrow-left.png" alt="arrow">
+            
+            </div>
+            <div class="lang-group">
+              <div class="lang">
+                <img src="./imgs/javascript.png" alt="javaScript">
+                <h6>javaScript</h6>
+              </div>
+              <div class="lang">
+                <img src="./imgs/html.png" alt="html">
+                <h6>HTML</h6>
+              </div>
+              <div class="lang">
+                <img src="./imgs/css.png" alt="css">
+                <h6>CSS</h6>
+              </div>
+            </div>
+          </div>
+          <div class="framewroks">
+            <h4>frameworks</h4>
+            <img class="arrow" src="./imgs/arrow-down.png" alt="arrow">
+          </div>
+          <div class="some-skill">
+            <h4>skills</h4>
+            <img class="arrow" src="./imgs/arrow-down.png" alt="arrow">
+          </div>
+        </section>
+    </section>
```

Comment:
- Please use the `ul` tag to group `languages`, `skills`, `frameworks`, each of them should be in an `li` tag. This will help improve your HTML semantics. 🙃 🙃 

- Also, use the `ul` tag to group each of the `languages`, each of them should be in their respective `li` tag. This is will improve your site's semantics and also improve accessibility. 😄 😄 

- There are also little discrepancies between the Figma design and your site. Please pay attention to the direction of the arrows, and case sensitivity for the letters. Sticking to the design as closely as possible gives confidence to recruiters in your ability to replicate exact Figma templates. 👍 👍 

Your design:
![skills project](https://user-images.githubusercontent.com/73607512/191774971-aca17b0b-0a58-4f9e-b043-6ff28668b0d1.PNG)

Figma template:
![skills figma](https://user-images.githubusercontent.com/73607512/191775069-db8ce983-34a3-4bd5-b566-c6f5434de7c8.PNG)